derive.cabal

```diff
@@ -38,7 +38,7 @@ library
         base == 4.*,
         filepath, syb, template-haskell, containers, pretty,
         directory, process, bytestring,
-        haskell-src-exts >= 1.18 && < 1.20,
+        haskell-src-exts >= 1.20,
```

Comment:
Please leave an upper bound - haskell-src-exts breaks something in almost every upgrade.

Reply:
I don't do that, because it is obvious nonsense :) No future version exists that breaks this packages, I simply do not know what to put here. In the future, I prefer the package breaks for a reason, not just because someone put arbitrary constraint on it.

Reply:
I very much agree, normally. However, for haskell-scr-exts, almost every single 0.1 bump in the last decade has broken both derive and hlint, so for haskell-src-exts I do have a fairly good idea of what the upper bound should be.

Reply:
How about `== 1.20.*` ?

Reply:
Yep, works for me (the only reason I managed to switch this to a range was that almost uniquely 1.18 and 1.19 could be simultaneously supported with only a small amount of adjustments.